app/lib/common/toolkit/errors/inline_error_button.dart

```diff
@@ -18,6 +18,7 @@ class ActerInlineErrorButton extends StatelessWidget {
 
   final String? dialogTitle;
   final String? text;
+  final Widget? child;
```

Comment:
allowing us to provide our own inner text to be shown for a better user experience.